sbnci/ICARUS/Modules/CMakeLists.txt

```diff
@@ -1 +1 @@
-add_subdirectory(PDS)
+add_subdirectory(JobConfigurations)
```

Comment:
My personal view is as you've put so much work into making the whole repo make a lot more sense organisationally we should also move:
```
ICARUS/Modules/JobConfigurations -> ICARUS/JobConfigurations
SBND/Modules/JobConfigurations -> SBND/JobConfigurations
```
As they aren't really modules and only live there because I stupidly put them there earlier in the year ;P

Reply:
Fair point. I mainly kept the Modules dir in case there were exp specific ana modules that could not be generalized, but I do agree the JobConfigurations dir doesn't need to be inside.